code/code/sys/connect.cc

```diff
@@ -2713,8 +2713,19 @@ void processAllInput()
           continue;
         } 
         if (IS_SET_DELETE(rc, DELETE_VICT)) {
-          delete d->character;
-          d->character = NULL;
+          if (!d->character) continue;
```

Comment:
Add a warning to log, perhaps?

Reply:
Are you meaning a warning for when `rc` ends up with the DELETE_VICT flag here but `d->character` is already null somehow?

Reply:
Yes. Is this supposed to happen? Probably not?